vscode/src/commands.ts

```diff
@@ -26,6 +26,8 @@ import {
   ChatMessageType,
   GetSolutionResult,
 } from "@editor-extensions/shared";
+import { KaiWorkflowMessage, KaiWorkflowMessageType } from "../../agentic/src/types";
```

Comment:
You should import from the actual project name instead of directly referencing the source file.  That way you're using the built output as a library and not pulling in the code directly.

Should look like:
```suggestion
import { KaiWorkflowMessage, KaiWorkflowMessageType } from "@editor-extensions/agentic";
```

You'll need to rebuild the agentic workspace anytime you make changes so it can be used by the other projects (there is no auto library cross compile in typescript/tsc/javascript/npm land)

Reply:
Yeah, this only exists so that i can debug easily with the debugger. Going to make it come from dist once pr is ready for review